marketplace/commercetools-sku/src/memoizePromise.js

```diff
@@ -0,0 +1,17 @@
+import memoize from 'lodash/memoize';
+
+function memoizePromiseResolver(...args) {
+  return JSON.stringify(args);
+}
+
+export function memoizePromise(f, resolver = memoizePromiseResolver) {
```

Comment:
I didn't catch exactly every instance of use but do we actually want to do this? This might mean that some editors will have to fully refresh the page to get new results in certain cases as data on commerce tools updates in real time.

Reply:
I removed it to be safe